locale/en/LC_MESSAGES/django.po

```diff
@@ -60,28 +102,197 @@ msgstr "Partner/Sponsor name"
+#, python-brace-format
+msgid "Descripteur déjà défini pour {prodcat}"
+msgstr "Descriptor already defined for {prodcat}"
+
+#: insalan/payment/models.py:28
+msgid "échouée"
+msgstr "failed"
+
+#: insalan/payment/models.py:29
+msgid "Réussie"
+msgstr "successful"
+
+#: insalan/payment/models.py:30
+msgid "En attente"
+msgstr "Pending"
+
+#: insalan/payment/models.py:31
+msgid "Remboursé"
+msgstr "Refunded"
+
+#: insalan/payment/models.py:37
+msgid "Inscription joueur⋅euse"
+msgstr "Player Registration"
+
+#: insalan/payment/models.py:38
+msgid "Inscription manager"
+msgstr "Manager Registration"
+
+#: insalan/payment/models.py:39
+msgid "Pizza"
+msgstr "Pizza"
+
+#: insalan/payment/models.py:46
+msgid "prix"
+msgstr "price"
+
+#: insalan/payment/models.py:48
+msgid "intitulé"
+msgstr "name"
+
+#: insalan/payment/models.py:49
+msgid "description"
+msgstr "description"
+
+#: insalan/payment/models.py:54
+msgid "Catégorie de produit"
+msgstr "Product category"
+
+#: insalan/payment/models.py:61
+msgid "Tournoi associé"
+msgstr "Associated tournament"
+
+#: insalan/payment/models.py:69
+msgid "Disponible à partir de"
+msgstr "Available starting on"
+
+#: insalan/payment/models.py:72
+msgid "Disponible jusqu'à"
+msgstr "Available until"
+
+#: insalan/payment/models.py:89
+msgid "Paiements"
+msgstr "Payments"
+
+#: insalan/payment/models.py:92
+msgid "Identifiant du paiement"
+msgstr "Payment identifier"
+
+#: insalan/payment/models.py:95 insalan/payment/models.py:360
+msgid "Transaction"
+msgstr "Transaction"
+
+#: insalan/payment/models.py:102 insalan/payment/models.py:153
+msgid "Montant"
+msgstr "Amount"
+
+#: insalan/payment/models.py:120
+msgid "Utilisateur"
+msgstr "User"
+
+#: insalan/payment/models.py:131
+msgid "État de la Transaction"
+msgstr "Transaction status"
+
+#: insalan/payment/models.py:133
+msgid "Date de creation"
+msgstr "Creation date"
+
+#: insalan/payment/models.py:135
+msgid "Date de dernière modification"
+msgstr "Last modification date"
+
+#: insalan/payment/models.py:141
+msgid "Identifiant du formulaire de paiement"
+msgstr "Intent checkout identifier"
+
+#: insalan/payment/models.py:146
+msgid "Identifiant de commande"
+msgstr "Order identifier"
+
+#: insalan/payment/models.py:170
+#, python-format
+msgid "Le produit %(id)s est actuellement indisponible"
+msgstr "Product %(id)s is currently unavailable"
+
+#: insalan/payment/models.py:177
+#, python-format
+msgid "Le tournoi %(id)s est actuellement indisponible"
+msgstr "Tournament %(id)s is currently unavailable"
+
+#: insalan/payment/models.py:200 insalan/payment/models.py:213
+msgid "Pas de handler trouvé pour un paiement"
+msgstr "No handler found for payment"
+
+#: insalan/payment/models.py:243
+#, python-format
+msgid "Transaction %(id)s en état invalide"
+msgstr "Transaction %(id)s in invalid state"
+
+#: insalan/payment/models.py:348
+msgid "Nombre d'un produit"
+msgstr "Product count"
+
+#: insalan/payment/models.py:349
+msgid "Nombres d'un produit"
```

Comment:
`Nombre de produits` ?

Reply:
More like `quantités d'un produit` tbh. It's the translation for `ProductCount`, which is just the medium model used for Many2Many between a transaction and products, so we can also store the amount of what products is bought.

I could instead have:
- singular: `Quantité d'un produit`
- plural: `Quantités de produits`

what do you think?

Reply:
Sounds way better

Reply:
Done